source/_integrations/oncue.markdown

```diff
@@ -30,3 +30,13 @@ The Oncue by Kohler integration will allow you to monitor the state of your [Onc
 - [KG80R](https://kohlerpower.com/en/residential/generators/product/kg80r)
 
 {% include integrations/config_flow.md %}
+
+## Non-functional and Incorrect Entities
+
+Different generators appear to support a different set of entities.  Examples for the 20RESA, without the engine heater option:  the engine coolant temperature sensor does not appear to actually exist, and it will show as 0C (32F), rather than e.g. `unavailable`.  Similarly, 0-ish values are shown for engine compartment temperature, current, all related (power) entities.   The total generated power entry shows an apparently fixed value that does not seem credible (18,285,792 kWh in one case, which is 18000 months of typical household usage!).   Similar behavior is seen for most of these entities on the official app.  With the engine heater option, at least some of these have useful values.
```

Comment:
```suggestion
Each generator supports a different set of sensors. With the engine heater option, some of these have unrealistic values. One example for the 20RESA, without the engine heater option: the engine coolant temperature sensor does not seem to exist, and it will show as 0C (32F), rather than, e.g. `unavailable`. Similarly, this case shows 0-ish values for engine compartment temperature, current, and all related (power) entities. The total generated power sensor may show an unexpectedly high value, but it generally increments at the expected rate. This behavior matches the official app.
```